src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -100,50 +84,11 @@ export const ConfirmSummary = ({
   const { priceImpactPercentage } = usePriceImpact(activeQuote)
   const walletSupportsBuyAssetChain = useWalletSupportsChain(buyAsset.chainId, wallet)
 
-  const userAddress = useMemo(() => {
-    if (!initialSellAssetAccountId) return ''
-
-    return fromAccountId(initialSellAssetAccountId).account
-  }, [initialSellAssetAccountId])
-
-  const { data: _isSmartContractSellAddress, isLoading: isSellAddressByteCodeLoading } =
```

Comment:
One too deep. We absolutely should *not* remove this, otherwise we'll introduce a regression and stop showing this copy 
<img width="522" alt="image" src="https://github.com/user-attachments/assets/7a725fe8-77ff-46e5-b53d-3f8965e61d82">

Swapping to a different EVM chain *should* be possible, only it should force manual receive addy entry.